src/screens.native.js

```diff
@@ -17,17 +17,17 @@ const getViewManagerConfigCompat = name =>
     ? UIManager.getViewManagerConfig(name)
     : UIManager[name];
 
-export function useScreens(shouldUseScreens = true) {
-  USE_SCREENS = shouldUseScreens;
-  if (USE_SCREENS && !getViewManagerConfigCompat('RNSScreen')) {
+export function enableScreens(shouldEnableScreens = true) {
```

Comment:
we should continue to export `useScreens` since there are a lot of places that will refer to it like this and it doesn't hurt to export it as well

Reply:
So, just for me to understand, you'd want to have both "names" exported? Meaning, you could then either use `useScreens` or `enableScreens`?

Reply:
yep

Reply:
ok - lemme tackle that now 👍